blockchain-albatross/src/blockchain.rs

```diff
@@ -92,7 +92,7 @@ pub struct Blockchain {
     pub network_id: NetworkId,
     // TODO network_time: Arc<NetworkTime>,
     pub notifier: RwLock<Notifier<'static, BlockchainEvent>>,
-    pub(crate) chain_store: Arc<ChainStore>,
+    pub chain_store: Arc<ChainStore>,
```

Comment:
Theoretically allows changing the database, which is not ideal. But then we also have an Arc to the database.

Reply:
What we could do is have all important objects wrapped into public facing types. This way we it's also easier to keep the public-facing API stable. We will still have fun with semantic versioning on all the other crates though.